lib/public/views/Runs/ActiveColumns/index.js

```diff
@@ -92,6 +93,21 @@ const activeColumns = (model) => ({
         size: 'w-10 f6 w-wrapped',
         format: (timestamp) => formatTimestamp(timestamp, false),
     },
+    runDuration: {
+        name: 'Duration',
```

Comment:
Even though it may be obvious, users would like to name it `Run Duration`

Reply:
Done